src/main/java/uk/co/ramp/covid/simulation/output/network/PeopleWriter.java

```diff
@@ -0,0 +1,23 @@
+package uk.co.ramp.covid.simulation.output.network;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+import uk.co.ramp.covid.simulation.population.Person;
+
+public class PeopleWriter {
+   
+    public static void writePeople(Appendable fileWriter, ArrayList<Person> allPeople) throws IOException {
+        String[] headers = {"id", "age"};
+        CSVPrinter csv = new CSVPrinter(fileWriter, CSVFormat.DEFAULT.withHeader(headers));
+
+        for (Person p : allPeople) {
+            csv.printRecord(p.getID(), p.getAge());
+        }
+        csv.close();
```

Comment:
I think we can lose the csv file handle before closing it here if the exception is thrown before we reach it?

In general we probably need to double check some of the File opening/closing locations. I think there's probably some places we can lose a handle (this can be done in another PR).

Reply:
You're right.  I don't think it's a big problem - I believe that if this fails, then the program will soon exit and Java should play nice and automatically close the file.  However, it would generally be a good idea to fix this and other places like it.